moabb/evaluations/evaluations.py

```diff
@@ -535,142 +515,121 @@ def evaluate(
     ):
         if not self.is_valid(dataset):
             raise AssertionError("Dataset is not appropriate for evaluation")
-        # Progressbar at subject level
-        results = []
-        with parallel_backend("threading"):
-            for result in Parallel(n_jobs=self.n_jobs_evaluation, verbose=1)(
-                delayed(self.process_subject)(
-                    subject,
-                    param_grid,
-                    pipelines,
-                    dataset,
-                    process_pipeline,
```

Comment:
@bruAristimunha here! this is why the `process_pipeline` argument is ignored now

Reply:
perfect! lets revert this part